democracylab/emails.py

```diff
@@ -379,3 +379,5 @@ def contact_democracylab_email(first_name, last_name, email_address, body):
         reply_to=[email_address]
     )
     send_email(email_msg)
+    
+    <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work by <a xmlns:cc="http://creativecommons.org/ns#" href="https://www.democracylab.org/" property="cc:attributionName" rel="cc:attributionURL">https://www.democracylab.org/</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```

Comment:
This line should go in democracylab/templates/emails/html_email_frame.html, preferably broken out into separate lines for readability